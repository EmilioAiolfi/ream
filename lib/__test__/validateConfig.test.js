const validateConfig = require('../validateConfig')

describe('options', () => {
  it('validate', () => {
    const res = validateConfig(true, {
      entry: 'hi',
      server: {
        port: 7000
      }
    })
    if (res.error) {
      throw res.error
    }
    expect(res.value).toMatchSnapshot()
  })
})
