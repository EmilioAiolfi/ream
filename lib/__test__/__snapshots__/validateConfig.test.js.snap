// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`options validate 1`] = `
Object {
  "css": Object {
    "extract": false,
  },
  "entry": "hi",
  "fsRoutes": Object {
    "baseDir": "pages",
    "basePath": "/",
    "match": /\\\\\\.\\(vue\\|js\\)\\$/i,
  },
  "generate": Object {
    "routes": Array [
      "/",
    ],
  },
  "minimize": false,
  "plugins": Array [],
  "productionSourceMap": true,
  "pwa": false,
  "runtimeCompiler": false,
  "server": Object {
    "host": "0.0.0.0",
    "port": 7000,
  },
  "transpileDependencies": Array [],
}
`;
